analytic_partner_hr_timesheet/models/account_analytic_line.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antiun Ingeniería S.L. - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class AnalyticAccountLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    @api.onchange('project_id')
+    def on_change_project_id(self):
+        if self.project_id:
```

Comment:
Do you mean write a method to set the partner of the account analytic linked to de project, when change the project ?